servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ClientEffectiveStrategyTest.java

```diff
@@ -564,6 +564,31 @@ void recordThread(final ClientOffloadPoint offloadPoint, final HttpExecutionStra
             });
         }
 
+        void flakyVerifyOffloads(ClientApi clientApi, HttpExecutionStrategy clientStrategy, String apiStrategy) {
+            // For flaky tests we see unexpected offloading at the Send stage. This is messing with CI so this
+            // particular test case is skipped for now.
+            // See https://github.com/apple/servicetalk/issues/2245
+            Throwable firstFlakyException = null;
+            Iterator<Throwable> it = errors.iterator();
+            while (it.hasNext()) {
+                Throwable t = it.next();
+                // Example message:
+                // Suppressed: java.lang.AssertionError: Expected IoThread or ForkJoinPool-1-worker-1
+                //  at Send, but was running on an offloading executor thread: client-executor-7-5.
+                //  clientStrategy=DEFAULT_HTTP_EXECUTION_STRATEGY, expectedStrategy=OFFLOAD_NONE_STRATEGY,
+                //  requestStrategy=DEFAULT_HTTP_EXECUTION_STRATEGY"
+                if (t.getMessage().contains("at Send, but was running on an offloading executor thread")) {
```

Comment:
Did you include "at Send" as a way to remove errors only for known flaky use-cases? If that was the only motivation, consider using `clientApi` parameter (and other if necessary) instead, because "at Send" can be used for other cases. Contains check can be focused only on "but was running on an offloading executor thread" part.